src/middleware/createCsrfErrorMiddleware.ts

```diff
@@ -1,22 +1,19 @@
-import { NextFunction, Request, Response } from 'express'
-import {
-    CsrfError
-} from '@companieshouse/web-security-node'
+import { NextFunction, Request, Response } from "express";
+import { CsrfError } from "@companieshouse/web-security-node";
 
 const csrfErrorTemplateName = "403";
 
 const csrfErrorHandler = (err: CsrfError | Error, _: Request,
   res: Response, next: NextFunction) => {
-  
-  // handle non-CSRF Errors immediately
+
   if (!(err instanceof CsrfError)) {
     return next(err);
   }
 
   return res.status(403).render(
     csrfErrorTemplateName, {
-        csrfErrors: true
-    }
+    csrfErrors: true
+  }
```

Comment:
these 2 lines are part of the json object passed through as a parameter to the render method call on line 13 and therefore should be indented further to show this

Reply:
thanks @hwilliams1 I have formatted the code.